proto/anki/deck_config.proto

```diff
@@ -152,6 +152,7 @@ message DeckConfig {
 
     // for fsrs
     float desired_retention = 37;
+    string ignore_before_date = 46;
```

Comment:
Please rename this and the references below to ignore_revlogs_before_date / ignore_revlogs_before_secs to make it clearer.